packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1835,36 +1835,48 @@ class EditableText extends StatefulWidget {
     required final VoidCallback? onCut,
     required final VoidCallback? onPaste,
     required final VoidCallback? onSelectAll,
+    required final VoidCallback? onLiveTextInput,
   }) {
-    // If the paste button is enabled, don't render anything until the state
-    // of the clipboard is known, since it's used to determine if paste is
-    // shown.
-    if (onPaste != null && clipboardStatus == ClipboardStatus.unknown) {
-      return <ContextMenuButtonItem>[];
+    final List<ContextMenuButtonItem> resultButtonItem = <ContextMenuButtonItem>[];
+
+    // Config button items with clip board.
+    if (onPaste == null || clipboardStatus != ClipboardStatus.unknown) {
```

Comment:
So Cut/Copy/Paste/Select All won't be shown until the clipboard status is known, but Live Text will show right away?  I guess that's ok, but just making sure that's a deliberate decision.

Reply:
> So Cut/Copy/Paste/Select All won't be shown until the clipboard status is known, but Live Text will show right away?

Yes. The Live Text function doesn't need clipboardStatus. So I think we should show live text button.